README.md

```diff
@@ -22,12 +22,12 @@ BKPR has been tested to work on a bare-minimum Kubernetes cluster with three kub
 
 The following matrix shows which Kubernetes versions and platforms are supported:
 
-| BKPR release | AKS versions  | GKE versions | EKS versions |
-|:------------:|:-------------:|:------------:|:-------------:
-|     `0.3`    | `1.8`-`1.9`   |      N/A     |      N/A     |
-|     `1.1`    | `1.9`-`1.10`  | `1.9`-`1.10` |      N/A     |
-|     `1.1`    | `1.10`-`1.11` | `1.11`       |      N/A     |
-|     `1.2`    | `1.10`-`1.11` | `1.11`       | `1.10`-`1.11`|
+| BKPR release   | AKS versions    | GKE versions   | EKS versions    |
+| :------------: | :-------------: | :------------: | :-------------: |
+| `0.3`          | `1.8`-`1.9`     | N/A            | N/A             |
+| `1.1`          | `1.9`-`1.10`    | `1.9`-`1.10`   | N/A             |
+| `1.1`          | `1.10`-`1.11`   | `1.11`         | N/A             |
+| `1.2`          | `1.11`-`1.12`   | `1.11-1.12`    | `1.11`          |
```

Comment:
Let's do this on 1.3 (which we can add now in master). So the change looks good, except this.

Reply:
fixed